### ISSUE TITLE
New window opens with blank server list

### ISSUE THREAD

**Issue by jakob** [jakob/Postico#827]
A customer has reported an issue where new windows open with a blank server list.

## What did you do?

- Connect to a server
- Press Cmd-N to open a new window

## What did you expect to happen?

- A new window should open with a list of servers

## What actually happened?

- The new window contains a blank list and shows servers only after pressing Cmd-R

## What software versions are you using?

Postico version:  
Build 9503 (latest dev build)

macOS version:  
13.0.1

PostgreSQL version:  
(probably not relevant)

**Comment by jakob** [comment]
I can't reproduce the problem on my machine.

The customer provided a video of the bug. On the video, there is also a layout bug visible in the server list when it works, so it could be that the problem is somehow related to layout.

**Comment by qwesda** [comment]
I noticed the same thing. I can reproduce it on my machine consistently. Glad to hear that CMD+R works ... I usually reopened postico. 

An additional observation: If you went trough the connect, new window, CMD+R cycle, connecting to the a server that already has an open window doesn't trigger the empty server list. Only the first window to a server does ...

Postico version: Build 9560
macOS version: 13.1 (22C65)

**Comment by jakob** [comment]
I've been trying to figure out what's going on by going through the source code, but I just can't figure out what the problem is.

I've made a test build with three buttons that reload different parts of the UI: https://downloads.eggerapps.at/postico/postico-9562.dmg

![image](https://user-images.githubusercontent.com/198305/219699749-7382d84d-b554-4528-a2bf-4222c17f94d9.png)

@qwesda If you can still reproduce the issue, could you try to download the build above and check which of the buttons fixes the bug?

**Comment by jakob** [comment]
Thank you, that narrows down my search.

**Comment by jakob** [comment]
I think I found the root cause of the bug. Build 9568 should fix it! A big thank you to everyone who reported the bug (and especially to you @qwesda for testing the test build). Sorry for having to endure this bug for two months.

In addition to fixing the bug, I've also added code that checks if the server list is empty after loading and automatically reloads the library if necessary, so hopefully this bug never happens again.
